pytorch_lightning/accelerators/accelerator_connector.py

```diff
@@ -350,7 +350,7 @@ def set_distributed_mode(self):
 
         rank_zero_info(f'GPU available: {torch.cuda.is_available()}, used: {self.trainer.on_gpu}')
         num_cores = self.trainer.tpu_cores if self.trainer.tpu_cores is not None else 0
-        rank_zero_info(f'TPU available: {XLA_AVAILABLE}, using: {num_cores} TPU cores')
+        rank_zero_info(f'TPU available: {XLADeviceUtils.tpu_device_exists()}, using: {num_cores} TPU cores')
```

Comment:
Why removing `XLA_AVAILABLE` ?

Reply:
`XLA_AVAILABLE` only checks if the XLA library is present. We need to ensure the TPU device exists on the machine :)

Reply:
we have defined not `TPU_AVALAIBLE` but to the point, that you have installed XLA does not mean you also TPU device